setup.py

```diff
@@ -12,7 +12,7 @@
 
 
 # Creating the version file
-version = '0.8.0a0'
+version = '0.8.1'
```

Comment:
This version shouldn't really matter since the version gets set by the git tag anyways through use of the `BUILD_VERSION` variable